pipeline.go

```diff
@@ -297,19 +297,8 @@ func (p *PipelineDefinition) Pipelines() (*Pipelines, error) {
 		steps := make(map[string]Step, 0)
 		// Verfiy steps and remove ignored from graph
```

Comment:
I think this comment might be wrong now

Reply:
I forgot to remove that one, the removal was there before.